src/utils/types/index.ts

```diff
@@ -35,6 +61,67 @@ export const gameSchema = z.object({
   videoTrailer: z.string().url().optional(),
 });
 
+export const gameSchemaAPI = z.object({
+  //Make sure to modify gameSchema endpoints, as well as editGameSchema I suppose.
+  name: z.string().min(3).max(50),
+  themes: z.array(z.string().refine(verifyObjectId)).optional(),
+  tags: z.array(z.string().refine(verifyObjectId)).optional(),
+  description: z.string(),
+  game: z.string().url(),
+  lesson: z.string().url().optional(),
+  parentingGuide: z.string().url().optional(),
+  answerKey: z.string().url().optional(),
+  videoTrailer: z.string().url().optional(),
+});
```

Comment:
What is the purpose of this schema?

Reply:
The original gameSchema was altered and then used for a different purpose, so I created a new version called gameSchemaAPI that I'm using specifically for parsing at the game endpoints.